CHANGELOG.md

```diff
@@ -9,14 +9,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Installer: Use init config.js from config.js.sample.
 - Switched out `rrule` package for `rrule-alt` in order to improve calendar issues. (Experimental: [#565](https://github.com/MichMich/MagicMirror/issues/565))
-- Make mouse events pass through the region fullscreen_above to modules below.
+- Make mouse events pass through the region fullscreen\_above to modules below.
```

Comment:
The  Pull Request isnt refect the fix and is already merged. I pushed a new #621